### PR TITLE
Create ur.js

### PR DESCRIPTION
**Pull request by abusarimhindi**
Adding Urdu language.
Requesting to increase number of words for sentence collector to get a meaningful sentence.
Urdu number have been added to the digits detector RegEx.

### CODE REVIEW

File: shared/validation/languages/ur.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import tokenizeWords from 'talisman/tokenizers/words';
+
+// Minimum of words that qualify as a sentence.
+const MIN_WORDS = 1;
+
+// Maximum of words allowed per sentence to keep recordings in a manageable duration.
+const MAX_WORDS = 25;
```
Comment (MichaelKohler) [inline shared/validation/languages/ur.js]: How long does it approximately take to read out 25 words aloud?

Comment (abusarimhindi) [reply shared/validation/languages/ur.js]: 8 to 10 seconds approx

**Comment by MichaelKohler** [comment]
Thanks, I will see if I can find somebody to review this.

**Comment by MichaelKohler** [comment]
I've asked for help in the Mozillians Telegram group.

Questions that I'm interested in:

* Does the maximum word setting make sense? Do you agree that this would take approximately 8 to 10 seconds to read out aloud?
* Is the numbers regex complete?
* Is the symbols regex complete? Are there any other symbols that should be filtered out?

**Comment by naeem9** [comment]
> * Does the maximum word setting make sense? Do you agree that this would take approximately 8 to 10 seconds to read out aloud?
> 
Yes.
> * Is the numbers regex complete?
> Yes , it is complete.
> * Is the symbols regex complete? Are there any other symbols that should be filtered out?
No ,following two ligatures should be excluded as these are itself phrases. ﷽  ﷺ

**Comment by abusarimhindi** [comment]
> I think initially MAX_WORDS = 20 is reasonable choice. 25 words may take may take 15 seconds.

No, it would not take 15 seconds because many joining words like (کو، سے، تک، اگر، اور، etc.) are single syllable words, and it's used mostly in Urdu sentences, and sir you also know that most of the Urdu words are single or bi-syllable.
I checked by speaking 25 word sentences loudly, it is taking a maximum 8 to 10 seconds. You may also check this. If we limit 20 words, it may be tough for collecting sentences.

**Comment by saif-u** [comment]
> I've asked for help in the Mozillians Telegram group.
> 
> Questions that I'm interested in:
> 
> * Does the maximum word setting make sense? Do you agree that this would take approximately 8 to 10 seconds to read out aloud?
> * Is the numbers regex complete?
> * Is the symbols regex complete? Are there any other symbols that should be filtered out?

» * It takes 8 to 10 seconds for 25 words to speak.

File: shared/validation/languages/ur.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+// Some languages want to check the structure, this is what this REGEX is for. For English
+// (and by extend as default) we're not currently using it.
+/* eslint-disable-next-line no-unused-vars */
+const STRUCTURE_REGEX = undefined;
+
+// The following symbols are disallowed, please update here as well and not just the regex
+// to make it easier to read:
+// < > + * \ # @ ^ [ ] ( ) ؄ ؃ ؂ ؏ ؆ ؇ ؈ ؁ ٪ ؉ ؊ ؔ ﷽ ؍ ؑ ؐ ﷺ ؒ ﷻ ۘ ۙ ۜ ۚ ۗ ۖ  ٝ ۢ !/ 
+/* eslint-disable-next-line no-useless-escape */
+const SYMBOL_REGEX = /[<>\+\*\\#@\^\[\]\(\)\؄\؃\؂\؏\؆\؇\؈\؁\٪\؉\؊\ؔ\﷽\؍\ؑ\ؐ\ﷺ\ؒ\ﷻ\ۘ\ۙ\ۜ\ۚ\ۗ\ۖ\٘\ٝ\ۢ\!\/]/;
```
Comment (abusarimhindi) [inline shared/validation/languages/ur.js]: I want to remove some symbol from here. How can I do this? I didn't find edit option.

**Pull request merged**